python/GafferUI/FileMenu.py

```diff
@@ -92,8 +92,11 @@ def __open( currentScript, fileName ) :
 	script = Gaffer.ScriptNode()
 	script["fileName"].setValue( fileName )
 
-	messageWidget = GafferUI.MessageWidget()
-	with messageWidget.messageHandler() :
+	with GafferUI.ErrorDialogue.ErrorHandler(
+		title = "Errors Occurred During Loading",
+		closeLabel = "Oy vey",
```

Comment:
Do we want to take this opportunity to replace the cheeky "Oy vey" with something more universal? Or are people happy sticking to Yiddish idioms?